build.gradle

```diff
@@ -233,7 +233,7 @@ dependencies {
     implementation 'com.github.hmcts:properties-volume-spring-boot-starter:0.1.1'
     implementation 'commons-io:commons-io:20030203.000550'
     implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.72'
-    implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.72'
+    implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.75'
```

Comment:
Should we keep the two tomcats in sync: `tomcat-embed-websocket` `tomcat-embed-core`?